lib/lita/handlers/locker_misc.rb

```diff
@@ -48,9 +48,18 @@ def log(response)
 
       def status(response)
         name = response.match_data['label']
-        return response.reply(status_label(name)) if Label.exists?(name)
-        return response.reply(status_resource(name)) if Resource.exists?(name)
-        response.reply(failed(t('subject.does_not_exist', name: name)))
+        unless name =~ /\*/
```

Comment:
AFAIK `=~` doesn't perform assignment on the left hand side variable. [Source](http://ruby-doc.org/core-2.2.0/Regexp.html#method-i-3D-7E)

```
>> foo = 'foo'
=> "foo"
>> foo =~ /o/
=> 1
>> foo
=> "foo"
```